homeassistant/components/mailbox/asterisk_cdr.py

```diff
@@ -0,0 +1,64 @@
+"""
+Asterisk CDR interface.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/mailbox.asteriskvm/
+"""
+import logging
+import hashlib
+import datetime
+
+from homeassistant.core import callback
+from homeassistant.components.asterisk_mbox import SIGNAL_CDR_UPDATE
+from homeassistant.components.asterisk_mbox import DOMAIN
```

Comment:
This platform belongs to the mailbox domain. Import the asterix_mbox domain as another name.
```py
from homeassistant.components.asterisk_mbox import DOMAIN as ASTERIX_MBOX_DOMAIN
```